### PR TITLE
move back graduation

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3412/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/ce714806d0dc760ff3bac29a6072bc8fe452d5e7/

File: content-src/experiments/notes.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id: 15
 title: 'Notes'
 slug: notes
-is_featured: true
+is_featured: false
```
Comment (chuckharmston) [inline content-src/experiments/notes.yaml:4]: Was this intentional?

Comment (johngruen) [reply content-src/experiments/notes.yaml:4]: yeah

**Review by chuckharmston — APPROVED** [review]

**Comment by chuckharmston** [comment]
@SoftVision-PaulOiegas would you mind double-checking dev tomorrow? We'll no longer have a featured experiment, which I don't think has happened since we shipped that feature.

**Comment by SoftVision-PaulOiegas** [comment]
@chuckharmston Tried to test this change on dev server but it seems that something went wrong with the merge. Tried also to rerun the job for it on CIrcleCI, but failed again.

However, I have managed to test this partially on local server and there doesn't seem to be any issues, at a short glance over the UI. I will double-check when it reaches dev properly.

txp no featured

**Pull request merged**